src/backend/dx12/src/device.rs

```diff
@@ -1045,8 +1049,8 @@ impl d::Device<B> for Device {
             image::Kind::D2(..) => {
                 desc.ViewDimension = winapi::D3D12_RTV_DIMENSION_TEXTURE2D;
                 *unsafe { desc.Texture2D_mut() } = winapi::D3D12_TEX2D_RTV {
-                    MipSlice: 0,
-                    PlaneSlice: 0,
+                    MipSlice: mip_level,
+                    PlaneSlice: layers.start,
```

Comment:
Plance slice is not the layer (https://msdn.microsoft.com/en-us/library/windows/desktop/dn705766(v=vs.85).aspx)

Reply:
whoa, great catch! TIL